cmd/app.go

```diff
@@ -59,7 +60,13 @@ var importAppCmd = &cobra.Command{
 
 	Run: func(ccmd *cobra.Command, args []string) {
 		appPath := args[0]
-		appID := internal.ImportApp(appPath, viper.GetString("engine"), headers)
+		engine := internal.ParseEngineURL(viper.GetString("engine"))
+		appID, appName, err := rest.ImportApp(appPath, engine, headers)
+		if err != nil {
+			internal.FatalError(err)
+		}
+		// TODO: Do we want to parse to map to host specifically or just the engine property?
```

Comment:
This is a good point, I think we should pass the parsed host instead. We should also make this change now (if we want it) cause it might lead to problems with existing `knownApps.yml` files e.g. not mapping IDs to app name of known hosts.

Reply:
Yes, we can fix it soon with #338